internal/exec/resolvable/resolvable.go

```diff
@@ -347,6 +347,13 @@ func unwrapNonNull(t common.Type) (common.Type, bool) {
 	return t, false
 }
 
+func stripGetPrefixAndUnderscore(s string) string {
+	if strings.HasPrefix(s, "Get") {
```

Comment:
should check if next field is "_" or `IsUpper` otherwise it would greedily match words starting with Get.